ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -146,6 +146,23 @@
           {{ tooltipTxtPdf }}
         </div>
       </v-tooltip>
+      <v-tooltip
+        v-else-if="!isDraft(item) && item.registeringName"
```

Comment:
hey Steph, instead of adding a new tooltip this would be a lot nicer if you just make *tooltipTxtPdf* a computed with this condition / new text. What do you think?